processor/configuration/src/main/resources/schemas/sources/aws_sqs_source_0.1.json

```diff
@@ -110,10 +110,7 @@
       "title": "Protocol",
       "description": "The underlying protocol used to communicate with SQS",
       "type": "string",
-      "enum": [
-        "http",
-        "https"
-      ],
+      "example": "http or https",
```

Comment:
Could you please point me the the connectors repository, I would like to double check this change. I tend to think I made there an update to have  `enums` instead of `exmaple`?

Reply:
https://github.com/bf2fc6cc711aee1a0c2a/cos-fleet-catalog-camel/blob/b0b652984487405e28767ecb90993aa322edc085/etc/kubernetes/manifests/base/connectors/connector-catalog-camel-aws/aws_sqs_source_0.1.json#L109